docs/pages/user-manual.mdx

```diff
@@ -458,10 +419,10 @@ tsh join <session_ID>
 ## Connecting to SSH clusters behind firewalls
```

Comment:
I think this should go into the security architecture article along with several Bastion, Jumphost, and firewall-related items.